src/streams/CreateStreamScreen.js

```diff
@@ -38,7 +38,7 @@ export default function CreateStreamScreen(props: Props): Node {
       }
 
       try {
-        await api.createStream(auth, { name, description, invite_only });
+        api.createStream(auth, { name, description, invite_only: privacy === 'private' });
```

Comment:
This lost the `await`, which means the try/catch will have no effect. Perhaps unintentional from the rebase?

Reply:
Thanks for the catch!